frontend/graph/conversions.go

```diff
@@ -59,30 +59,28 @@ func instrumentedApplicationToActualSource(instrumentedApp v1alpha1.Instrumented
 		})
 	}
 
-	// Map the conditions of the application
+	// Map the conditions
 	var conditions []*model.Condition
-	for _, condition := range instrumentedApp.Status.Conditions {
+	for _, condition := range instruConfig.Status.Conditions {
 		conditions = append(conditions, &model.Condition{
-			Type:               condition.Type,
 			Status:             k8sConditionStatusToGql(condition.Status),
+			Type:               condition.Type,
 			Reason:             &condition.Reason,
-			LastTransitionTime: k8sLastTransitionTimeToGql(condition.LastTransitionTime),
 			Message:            &condition.Message,
+			LastTransitionTime: k8sLastTransitionTimeToGql(condition.LastTransitionTime),
 		})
 	}
 
 	// Return the converted K8sActualSource object
 	return &model.K8sActualSource{
-		Namespace:         instrumentedApp.Namespace,
-		Kind:              k8sKindToGql(instrumentedApp.OwnerReferences[0].Kind),
-		Name:              instrumentedApp.OwnerReferences[0].Name,
-		ServiceName:       &instrumentedApp.Name,
+		Namespace:         instruConfig.Namespace,
+		Kind:              k8sKindToGql(instruConfig.OwnerReferences[0].Kind),
+		Name:              instruConfig.OwnerReferences[0].Name,
```

Comment:
what is the difference between name, serviceName, and reportedName?

Reply:
Name is the app name we show in the UI (eg. `load-generator`).
ServiceName is the name with "kind" is prefix (eg `deployment-load-generator`).
ReportedName is the name given on user-update (eg `test 1234`).

Reply:
I dont think we need the service name in the ui